src/main/java/biz/paluch/logging/gelf/intern/sender/GelfTCPSender.java

```diff
@@ -64,13 +71,25 @@ public GelfTCPSender(String host, int port, int connectTimeoutMs, int readTimeou
     public GelfTCPSender(String host, int port, int connectTimeoutMs, int readTimeoutMs, int deliveryAttempts,
             boolean keepAlive, ErrorReporter errorReporter) throws IOException {
 
+        this(host, port, connectTimeoutMs, readTimeoutMs, deliveryAttempts, keepAlive, 50, 10, connectTimeoutMs, errorReporter);
+    }
+
+    public GelfTCPSender(String host, int port, int connectTimeoutMs, int readTimeoutMs, int deliveryAttempts,
```

Comment:
It would make sense to encapsulate backoff with an interface (`BackOff.start()` returning `BackOffExecution`, `BackOffExecution.nextBackOff()` would return the time in millis that the execution should wait) so we can provide a more flexible (constant/exponential) backoff implementation without having a growing constructor.